src/common/types.cpp

```diff
@@ -1740,4 +1740,44 @@ bool LogicalType::operator==(const LogicalType &rhs) const {
 	return EqualTypeInfo(rhs);
 }
 
+bool LogicalType::Contains(const LogicalTypeId other) const {
+	switch (id()) {
+	case LogicalTypeId::STRUCT: {
```

Comment:
Can we use the `TypeVisitor` here?

Reply:
Oh, amazing, I was looking for something like this.